src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -151,4 +164,20 @@ public EventTypeOptions getOptions() {
     public void setOptions(final EventTypeOptions options) {
         this.options = options;
     }
+
+    public Set<String> getWriteScopes() {
+        return writeScopes;
```

Comment:
I think as a step to immutability it can be wrapped with `Collections.unmodifiableSet(...)`